### ISSUE TITLE
-inputPath is ignored when reading CSV input files

### ISSUE THREAD

**Issue by casella** [OpenModelica/OpenModelica#9482]
Consider this simple test model with top level inputs
```
model TestTopInputs
  Modelica.Blocks.Interfaces.RealInput u1;
  Modelica.Blocks.Interfaces.RealInput u2;
  Modelica.Blocks.Interfaces.RealInput u3;
  Real y = 2*u1;
  Real z = 4*u2;
  Real w = u1*u2*u3;
  annotation(
    uses(Modelica(version = "4.0.0")),
    experiment(StartTime = 0, StopTime = 5, Tolerance = 1e-06, Interval = 0.01));
end TestTopInputs;
```
My OMEdit working directory is `D:\Temp\OMEdit.` If I load this model in OMEdit, put a file abc.csv with the following content:
```
time, u3, u2, u1,
0, 0, 0, 0,
1, 0, 0, 0,
2, 0, 0, 1,
3, 0, 2, 1,
4, 3, 1, 1,
5, 1, 1, 1,
```
in the `D:\Temp\OMEdit\TestTopInputs` directory, and then simulate it with the `-csvInput=abc.csv` simulation flag, I get 
```
Simulation process failed. Exited with code 0x00000001.
Failed to read CSV-file abc.csv
```
despite OMEdit setting 
```
-inputPath=D:/Temp/OMEdit/TestTopInputs
```
Something similar happens if I set -exInputFile=abc.csv
```
OMC can't find the file abc.csv.
```
However, if I put the `abc.csv` file in the same directory where I put the `TestTopInput.mo` file (in this case `D:/Temp/OMEdit`) then the simulations with both flags can find the file.

The path set by -inputPath should also be considered in both cases. 

For -exInputFile, I understand the place where the directory should be added is [here](https://github.com/OpenModelica/OpenModelica/blob/14823c56f2dd1a807355faf547ecb966f61d123a/OMCompiler/SimulationRuntime/c/simulation/solver/external_input.c#L66), and [here](https://github.com/OpenModelica/OpenModelica/blob/14823c56f2dd1a807355faf547ecb966f61d123a/OMCompiler/SimulationRuntime/c/simulation/solver/external_input.c#L70) if the default file name externalInput.csv is used.

For -csvInput, I understand the place is [here](https://github.com/OpenModelica/OpenModelica/blob/master/OMCompiler/SimulationRuntime/c/simulation/solver/external_input.c#L104).

It's not too clear to me what is the difference between using -csvInput and using -exInputFile , as this is not documented anywhere (I'm trying to do this myself). Apparently, the former uses libcsv while the latter doesn't, I have no clue what this entails, but this is another story.

**Comment by casella** [comment]
@AnHeuermann could you please shed some light, and possibly fix this? Adding the input path to the filename shouldn't be a big deal.

**Comment by mahge** [comment]
I can take a look and see if we can fix this and the other issues (#9486, #9487, and #9488).
